eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/ReplicationEndpoint.scala

```diff
@@ -509,11 +532,13 @@ private class Connector(sourceConnector: SourceConnector) extends Actor {
     case GetReplicationEndpointInfoSuccess(info) if !connected =>
       sourceConnector.links(info).foreach {
         case ReplicationLink(source, target) =>
-          val filter = sourceConnector.connection.filters.get(target.logName) match {
-            case Some(f) => f
-            case None    => NoFilter
+          if (sourceLogIds.forall(_.contains(source.logId))) {
```

Comment:
Why not having a `replicationLinks: Set[ReplicationLink]` parameter instead if callers want to control which replicators should be set up? All callers have access to the `SourceConnector.links` method to obtain the links they need.